packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -280,11 +294,22 @@ export abstract class QueueProcessingServiceBase extends Construct {
    * The scaling interval for autoscaling based off an SQS Queue size.
    */
   public readonly scalingSteps: ScalingInterval[];
+
   /**
    * The AwsLogDriver to use for logging if logging is enabled.
    */
   public readonly logDriver?: LogDriver;
 
+  /**
+   * Flag to disable CPU based auto scaling strategy on the service.
+   */
+  readonly disableCpuBasedScaling: boolean;
+
+  /**
+   * The target CPU utilization percentage for CPU based scaling strategy when enabled.
+   */
+  readonly cpuTargetUtilizationPercent: number;
```

Comment:
```suggestion
  private readonly cpuTargetUtilizationPercent: number;
```

@AnuragMohapatra I'm pretty sure this would still default to `public`, and at any rate, it's good practice to always be explicit about the scope